assisted-inject-processor-dagger2/src/test/java/com/squareup/inject/assisted/dagger2/processor/AssistedInjectDagger2ProcessorTest.kt

```diff
@@ -368,6 +368,26 @@ class AssistedInjectDagger2ProcessorTest {
         .`in`(moduleOne).onLine(7)
   }
 
+  @Test fun nestedModule() {
+    val module = JavaFileObjects.forSourceString("test.TestModule", """
+      package test;
+
+      import com.squareup.inject.assisted.dagger2.AssistedModule;
+      import dagger.Module;
+
+      class Outer {
+          @AssistedModule
+          @Module(includes = AssistedInject_TestModule.class)
+          public abstract class TestModule {}
+      }
+    """)
+
+    assertAbout(javaSource())
+      .that(module)
+      .processedWith(AssistedInjectDagger2Processor())
+      .compilesWithoutError() //TODO should it compile or be a specific error prohibiting nesting?
```

Comment:
Remove TODO, add to `@Ignore` on method, add expected generated code. This way we can merge the test and land the fix separately.